public/data/saifrahman.json

```diff
@@ -0,0 +1,44 @@
+{
+    "name": "Saif Rahman",
+    "type": "personal",
+    "bio": "Engineer-Student-Enthusiast",
+    "avatar": "https://avatars.githubusercontent.com/u/64410006?s=400&u=bb5179ef3d88c859be1907b535cbc6b1111372e6&v=4",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/saifrahmn",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/saif_rahmn",
+        "icon": "twitter"
+      },
+      {
+        "name": "Read my Blogs",
+        "url": "https://saifrahman.hashnode.dev",
+        "icon": "Hashnode"
+      },
+      {
+        "name": "Follow me on Instagram",
+        "url": "https://www.instagram.com/saif_rahm.n",
+        "icon": "Instagram"
+      },
+      {
+        "name": "Visit my Portfolio",
+        "url": "",
```

Comment:
You can remove the portfolio section if you don't have one 😅

Reply:
@saifrahmn Please mark this change as resolved.